### ISSUE TITLE
Porting xstream-boilerplate to create-cycle-app flavor

### ISSUE THREAD

**Issue by nickbalestra** [cyclejs-community/create-cycle-app#32]
As per discussion on https://github.com/cyclejs-community/xstream-boilerplate/issues/233

**Comment by geovanisouza92** [comment]
I was thinking, as xstream-boilerplate is, as name suggests, xstream-focussed, this could raise the idea about limiting flavors by its supported stream-libs, otherwise, this will force flavors to support multiple stream libraries.

For the core flavors, this is a must, but for private/personal flavors that could be used by anyone, this could be annoying, considering that, if I prefer to use one library for my projects, this would be the preferred one for all projects.

**Comment by Widdershin** [comment]
I would also quite like to limit my flavor to xstream only.

**Comment by nickbalestra** [comment]
Yes, I totally agree. We should have a way to opt-out from streamlib selection, in the case that a flavor only want to support one (it makes a lot of sense indeed for custom flavors)
